api/lib/domain/models/ComplementaryCertificationScoringWithComplementaryReferential.js

```diff
@@ -20,10 +21,15 @@ class ComplementaryCertificationScoringWithComplementaryReferential extends Part
     this.reproducibilityRate = reproducibilityRate;
     this.hasAcquiredPixCertification = hasAcquiredPixCertification;
     this.minimumReproducibilityRate = minimumReproducibilityRate;
+    this.isRejectedForFraud = isRejectedForFraud;
   }
 
   isAcquired() {
-    return this.hasAcquiredPixCertification && this.reproducibilityRate.isEqualOrAbove(this.minimumReproducibilityRate);
+    return (
+      !this.isRejectedForFraud &&
+      this.hasAcquiredPixCertification &&
```

Comment:
hors scope: je me demande pourquoi on a pas `this.hasAcquiredPixCertification` dans les 2 cas (`ComplementaryCertificationScoringWithComplementaryReferential` & `ComplementaryCertificationScoringWithoutComplementaryReferential`)

Reply:
C'est vrai qu'il n'y aurait pas eu le bug. 
C'est possible d'obtenir clea sans obtenir Pix ? On peut en parler avec Agnès demain au morning je pense.

Reply:
Suite à un long échange avec Agnès, les termes métiers nous amènent à penser qu'il faut conserver la logique de rejet pour fraude de façon explicite.
Et qu'il est aussi juste de dire que une `ComplementaryCertificationScoringWithoutComplementaryReferential` ne peut être obtenue sans avoir acquis la certification Pix coeur.
J'ajoute donc le flag d'obtention de pix coeur dans la condition d'acquisition de la complémentaire sans référentiel dans un deuxième commit